src/appleseed/renderer/modeling/project/project.xsd

```diff
@@ -294,22 +294,6 @@
                         </xsd:complexContent>
                     </xsd:complexType>
                 </xsd:element>
-                <xsd:element name="rules" minOccurs="0">

```

Comment:
I don't think we can load them in any case, no? I removed the code that loads them.
I'm not very familiar with Xerces. What happens if we don't handle an element?